validators.go

```diff
@@ -686,14 +686,63 @@ func (v *validator) isOverrideIndicator(code string) error {
 	return errors.New(msg)
 }
 
-// isUpperAlphanumeric checks if string only contains ASCII alphanumeric upper case characters
+// isImageViewAnalysisValid ensures generic properties of imageViewAnalysis are valid
+func (v *validator) isImageViewAnalysisValid(code int) error {
+	switch code {
+	case
+		// Refer to ImageViewAnalysis property
+		0,
+		// Refer to ImageViewAnalysis property
+		1,
+		// Refer to ImageViewAnalysis property
+		2:
+		return nil
+	}
+	return errors.New(msgImageViewAnalysisValid)
+}
+
+// Returns
+
+// isReturnNotificationIndicator ensures ReturnNotificationIndicator of ReturnDetail is valid
+func (v *validator) isReturnNotificationIndicator(code int) error {
+	switch code {
+	case
+		// Preliminary notification
+		1,
+		// Final notification
+		2:
+		return nil
+	}
+	msg := fmt.Sprintf(msgInvalid, "ReturnNotificationIndicator")
+	return errors.New(msg)
```

Comment:
I usually use `fmt.Errorf(msg, args...)` for building error messages.

Edit: Should we include the value on these errors?

Reply:
Added a ToDo in validators for that.